@here/harp-datasource-protocol/lib/operators/InterpolationOperators.ts

```diff
@@ -279,11 +279,31 @@ function prepareStepCallExpr(call: StepCallExpr) {
     }
 }
 
-const operators = {
+const operators: OperatorDescriptorMap = {
     interpolate: {
         isDynamicOperator: (call: CallExpr): boolean => {
             return call.args[1] && call.args[1].isDynamic();
         },
+        partialEvaluate: (context: ExprEvaluatorContext, call: InterpolateCallExpr): Value => {
+            prepareInterpolateCallExpr(call);
```

Comment:
Looks like not needed, in partial evaluation context we don't even try to use interpolators.
(same applies to second call below)

Reply:
in partial evaluating we need the context because we are resolve feature properties, e.g ["get", and so on.. what we keep unchanged when partial evaluating is dynamic expressions.

Reply:
I'm introducing InterpolateExpr and StepExpr so i can remove the call to the prepare methods